config/environments/production.rb

```diff
@@ -60,18 +60,9 @@
   # Use a different cache store in production
   config.cache_store = :memory_store
 
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server
-  if ENV['ASSET_SYNC_ENABLED'] == 'false'
-     # noop
-  elsif ENV['CLOUDFRONT_HOST']
+  # Enable serving of images, stylesheets, and JavaScripts from Cloudfront
+  if ENV["CLOUDFRONT_HOST"]
```

Comment:
We're still using cloudfront. I don't want to depend exclusively on cloudflare since we don't have to.